app/views/agents/rdvs/index.html.slim

```diff
@@ -1,7 +1,37 @@
+- content_for :title do
+  | Liste des RDV
+
 - content_for :breadcrumb do
   ol.breadcrumb.m-0
-    li.breadcrumb-item
-      = link_to 'Vos statistiques', organisation_agent_stats_path(current_organisation, current_agent)
-    li.breadcrumb-item.active Liste des RDV
+    - if @form.date
+      li.breadcrumb-item
+        = link_to "Agenda de #{@agent.full_name}", organisation_agent_path(current_organisation, @agent, date: @form.date)
+    - else
+      li.breadcrumb-item
+        = link_to 'Vos statistiques', organisation_agent_stats_path(current_organisation, current_agent)
+      li.breadcrumb-item.active Liste des RDV
+
+.row.justify-content-md-center
+  .col-md-6
+    .card.border-info
+      .card-header ⚙️ Filtres et options
+      .card-body
+        = simple_form_for(@form, method: "GET", url: url_for({}), as: "") do |f|
+          = f.input :agent_id, collection: [@agent], label: "Agent", label_method: :full_name, input_html: { class: 'select2-input', disabled: true }, wrapper: "horizontal_form"
```

Comment:
`disabled: true` t'es pas sympa avec les agents ? 👺

Reply:
je l'ai noté dans la description de la PR, j'ai pas envie de trop me prendre la tete avec les policies donc j'ai pris le raccourci `collection: [@agent]` et `disabled: true`. Je dépasse deja pas mal le cadre de la fonctionalité demandée initialement (print today rdvs), donc je me suis dit qu'on pourrait permettre cette autre fonctionnalité (changer d'agent depuis cette liste de RDVs) plus tard.

Reply:
mais je veux quand meme afficher le nom de l'agent courant ici car je trouve la fonctionnalité cassée sans ça (tu vois une liste de RDVs sans aucun contexte). je l'affiche dans un select disabled pour clarifier l'intention future anticipée de permettre le changement